docs/user/03-21-instance-level-mapping.md

```diff
@@ -77,7 +89,9 @@ To create a service instance with a custom Secret, you must use the **btpAccessC
     You see the status `Created`.
```

Comment:
The output of the command is:
```
➜  btp-manager git:(add_missing_steps) ✗ kubectl get serviceinstances.services.cloud.sap.com test -o yaml
apiVersion: services.cloud.sap.com/v1
kind: ServiceInstance
metadata:
  creationTimestamp: "2024-10-25T13:01:10Z"
  finalizers:
  - services.cloud.sap.com/sap-btp-finalizer
  generation: 1
  name: test
  namespace: default
  resourceVersion: "21406"
  uid: f058b439-0d2c-4930-b9c5-209653698ad3
spec:
  btpAccessCredentialsSecret: my-secret
  externalName: test
  serviceOfferingName: auditlog-management
  servicePlanName: default
  userInfo:
    groups:
    - system:authenticated
    username: wojciech.wrzalik@sap.com
status:
  conditions:
  - lastTransitionTime: "2024-10-25T13:01:12Z"
    message: ServiceInstance provisioned successfully
    observedGeneration: 1
    reason: Created
    status: "True"
    type: Succeeded
  - lastTransitionTime: "2024-10-25T13:01:12Z"
    message: ServiceInstance provisioned successfully
    observedGeneration: 1
    reason: Provisioned
    status: "True"
    type: Ready
  hashedSpec: d5f49e0b55915f6ac18c66d65b2b213b
  instanceID: 1b592780-d873-4d96-a81f-624fcc5c55d2
  observedGeneration: 1
  ready: "True"
  subaccountID: b43118f2-bb21-496b-bcfc-a3d34719d6b6
```

Let's remove `-o json` option from the command example.